src/applications/vaos/components/review/ReasonForAppointmentSection.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router';
+import newAppointmentFlow from '../../newAppointmentFlow';
+import { PURPOSE_TEXT } from '../../utils/constants';
+
+export default function ReasonForAppointmentSection(props) {
+  return (
+    <>
+      <div className="vads-l-grid-container vads-u-padding--0">
+        <div className="vads-l-row">
+          <div className="vads-l-col--6">
+            <h3 className="vaos-appts__block-label">
+              {PURPOSE_TEXT[props.data.reasonForAppointment]} visit
```

Comment:
Adding visit at the end doesn't work when they choose "My reason is not listed here"